benchmarks/src/main/java/org/apache/druid/benchmark/query/SelectBenchmark.java

```diff
@@ -278,8 +278,9 @@ private SelectQuery incrementQueryPagination(SelectQuery query, SelectResultValu
     Map<String, Integer> pagingIdentifiers = prevResult.getPagingIdentifiers();
     Map<String, Integer> newPagingIdentifers = new HashMap<>();
 
-    for (String segmentId : pagingIdentifiers.keySet()) {
-      int newOffset = pagingIdentifiers.get(segmentId) + 1;
+    for (Map.Entry<String, Integer> entry : pagingIdentifiers.entrySet()) {
```

Comment:
It could be just
```java
pagingIdentifiers.replaceAll(id, offset) -> offset + 1);
```

Reply:
updated this

Reply:
`pagingIdentifiers.replaceAll((id, offset) -> offset + 1);` should replace the whole loop, not be done on every iteration.

@hpandeycodeit please read your code with fresh eye every time before committing.